bcipy/signal/evaluate/artifact.py

```diff
@@ -40,17 +42,19 @@ class DefaultArtifactParameters(Enum):
     """
 
     # Voltage
-    PEAK_THRESHOLD = 75e-7
-    PEAK_MIN_DURATION = 0.005
+    PEAK_THRESHOLD = 15e-6
+    PEAK_MIN_DURATION = 0.001
```

Comment:
It might be useful to have a comment or documentation somewhere about how these values were determined.

Reply:
I will add a comment; before we release 2.0, I am proposing a few enhancements to this module that should help with configuration and use. https://www.pivotaltracker.com/story/show/188483451